extensions/wikia/UserLogin/js/FacebookLogin.js

```diff
@@ -301,7 +305,10 @@
 
 			this.loginForm = new window.FacebookFormConnectUser($modal.find('.UserLoginFacebookRight'), {
 				ajaxLogin: true,
+				ajaxValidation: true,
 				skipFocus: true,
+				usernameInputName: 'userloginext01',
+				passwordInputName: 'userloginext02',
```

Comment:
Is there a chance to put these strings ('userloginext01' and 'userloginext02') into some common variable? Just to have it defined in one place.